terragrunt/aws/central_account/inputs.tf

```diff
@@ -0,0 +1,9 @@
+variable "billing_code" {
+  description = "The billing code to tag our resources with"
+  type        = string
+}
```

Comment:
This is getting handled by the global `billing_tag_value` we set in the root `terragrunt.hcl`.